make_mocks.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for experiment."""
+
+import numpy as np
+import utils
+from pathlib import Path
+
+test_set_dir = 'mock/test_sets'
+mock_test_set_filepath = Path(test_set_dir) / 'fakepdb' / 'test_set_1.npz'
+mock_mogwai_filepath = 'mock/fakepdb_model_state_dict.npz'
+L = 3
+A = 5
+
+
+def write_mock_mogwai_state_dict():
```

Comment:
I was originally considering this, but because they expect a certain directory structure `{test_dir}/{pdb_name}/*.npz` I thought it would be easier to set that up statically... Are you suggesting write to static directories and then delete, so that we don't have to merge the npz files into the repo?

Reply:
I think what you have is fine. Typically these directories are called something like ./test_data. Can you rename ./mock/ to ./test_data?